components/checkbox/stories/checkbox.stories.js

```diff
@@ -130,7 +130,152 @@ const CheckboxGroup = (args) => html`
+      })}
+    </div>
+    <div style="display: flex; flex-direction: column; padding: 16px">
+    	<h4>Invalid</h4>
+      ${Template({
+        ...args,
+        isInvalid: true,
+        iconName: undefined,
+      })}
+      ${Template({
+        ...args,
+        isInvalid: true,
+        isChecked: true,
+      })}
+      ${Template({
+        ...args,
+        isInvalid: true,
+        isIndeterminate: true,
+      })}
+        ${Template({
+        ...args,
+        isInvalid: true,
+        isDisabled: true,
+      })}
+      ${Template({
+        ...args,
+        isInvalid: true,
+        label: "Checkbox with an extraordinarily long label. Please don't do this but if you did, it should wrap text when it gets longer than the container that houses the checkbox with the unacceptably long label",
+        customStyles: { "max-inline-size": "200px" },
+      })}
+    </div>
+    <div style="display: flex; flex-direction: column; padding: 16px">
+    	<h4>Disabled</h4>
+      ${Template({
+        ...args,
+        isDisabled: true,
+        iconName: undefined,
+      })}
+      ${Template({
+        ...args,
+        isDisabled: true,
+        isChecked: true,
+      })}
+      ${Template({
+        ...args,
+        isDisabled: true,
+        isIndeterminate: true,
+      })}
+        ${Template({
+        ...args,
+        isDisabled: true,
+      })}
+      ${Template({
+        ...args,
+        isDisabled: true,
+        label: "Checkbox with an extraordinarily long label. Please don't do this but if you did, it should wrap text when it gets longer than the container that houses the checkbox with the unacceptably long label",
+        customStyles: { "max-inline-size": "200px" },
+      })}
+    </div>
+    <div style="display: flex; flex-direction: column; padding: 16px">
+      <h4>Read-only</h4>
+      ${Template({
+        ...args,
+        iconName: undefined,
+      })}
+      ${Template({
+        ...args,
+        isReadOnly: true,
+        isChecked: true,
+      })}
+      ${Template({
+        ...args,
+        isReadOnly: true,
+        isIndeterminate: true,
+      })}
+        ${Template({
+        ...args,
+        isReadOnly: true,
+        isDisabled: true,
+      })}
+      ${Template({
+        ...args,
+        isReadOnly: true,
+        label: "Checkbox with an extraordinarily long label. Please don't do this but if you did, it should wrap text when it gets longer than the container that houses the checkbox with the unacceptably long label",
+        customStyles: { "max-inline-size": "200px" },
+      })}
+    </div>
+  </div>
+`;
+
 export const Default = CheckboxGroup.bind({});
 Default.args = {
 	id: "default-checkbox",
 };
+
+/**
+ * Stories for the MDX "Docs" only.
+ */
+export const DefaultVariants = AllVariantsCheckboxGroup.bind({});
+DefaultVariants.tags = ["docs-only"];
+DefaultVariants.parameters = {
+	chromatic: { disableSnapshot: true }
+};
+
+export const Emphasized = CheckboxGroup.bind({});
```

Comment:
Do we want this single Emphasized checkbox group as a story in the sidebar at all? It's not on the production storybook right now, and people can toggle the "emphasized styling." Does it feel like it gets lost at all?

Reply:
Good question. As we're reorganizing things currently, we wouldn't have it as a separate story as we're trying to limit the number of stories. The Docs I think will help call attention to it.